plugin.json

```diff
@@ -21,12 +23,8 @@
 
     "dashboards" : [
         {
-            "name" : "Nginx Free",
+            "name" : "Nginx F/OSS",
             "layout" : "d-w=3&d-h=2&d-pad=5&d-bg=none&d-g-NGINX_ACTIVE_CONNECTIONS=0-0-1-1&d-g-NGINX_READING=2-1-1-1-t&d-g-NGINX_WRITING=2-1-1-1-b&d-g-NGINX_REQUESTS=1-0-1-1&d-g-NGINX_HANDLED=1-1-1-1-t&d-g-NGINX_NOT_HANDLED=1-1-1-1-b&d-g-NGINX_REQUESTS_PER_CONNECTION=2-0-1-1&d-g-NGINX_WAITING=0-1-1-1"
-        },
-        {
-            "name" : "Nginx Plus",
```

Comment:
Agree there are not many difference, but having empty graphs kind of suck, it makes me think something is broken.  The benefit of having two dashboards is at least you know you do not have the nginx plus version.

We should probably just undelete the dashboard and leave it the way it was in this case.